app/service.js

```diff
@@ -134,6 +144,9 @@ app.get('/__version__', function(req, res) {
   res.json(versionData);
 });
 
+// The error handler must be before any other error middleware
+app.use(raven.middleware.express.errorHandler(sentryDSN));
```

Comment:
Should we move this `raven` business up next to the `app.use(raven.middleware.express.requestHandler(sentryDSN));` on L87?

Plus, if I don't have a Sentry account/DSN, can I still run this server locally, or will it implode with the force of 10000 suns? Or in other words, should we have a way to pref this off for local development?